docs/elements.md

```diff
@@ -122,14 +122,14 @@ Gives automated feedback in the case of improper asymptotic input.
 | Attribute        | Type                                                  | Default                 | Description                                                                                                                                                          |
 | ---------------- | ----------------------------------------------------- | ----------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | `answers-name`   | string                                                | —                       | Variable name to store data in. Note that this attribute has to be unique within a question, i.e., no value for this attribute should be repeated within a question. |
-| `type`           | "big-o", "theta", "omega", "little-o", "little-omega" | "big-o"                 | Type of asymptotic answer required.                                                                                                                                  |
+| `type`           | `big-o`, `theta`, `omega`, `little-o`, `little-omega` | `big-o`                 | Type of asymptotic answer required.                                                                                                                                  |
```

Comment:
@nwalters512 I believe we want backticks with quotes, i.e. `"big-o"`? Do you have strong feelings on this?

Reply:
That's what I'm leaning towards, yeah.